lib/11/libLLVM_h.jl

```diff
@@ -4552,7 +4552,7 @@ end
     LLVMDWARFEmissionLineTablesOnly = 2
 end
 
-@cenum __JL_Ctag_204::UInt32 begin
+@cenum LLVMMetadataKind::UInt32 begin
```

Comment:
@Gnimuc This is unfortunate, but necessary, as we pass these enum values as a `Val` in which case they need to match exactly (and the equivalence to a UInt32 doesn't matter).

Reply:
I'll try to update the wrapper generator script in compliance with this change in #245.